artifactory/commands/transferfiles/srcpluginservice.go

```diff
@@ -34,19 +35,22 @@ func (sup *srcUserPluginService) IsDryRun() bool {
 	return false
 }
 
-func (sup *srcUserPluginService) getUploadChunksStatus(ucStatus UploadChunksStatusBody) (UploadChunksStatusResponse, error) {
+func (sup *srcUserPluginService) syncChunks(ucStatus *UploadChunksStatusBody) (UploadChunksStatusResponse, error) {
```

Comment:
* Let's send the `ucStatus` argument by value and not by reference, to reduce the concern that this function will modify it.
* Let's remive the following clean up lines, and let the caller function handle the cleanup - 
```
	ucStatus.AwaitingStatusChunks = nil
	ucStatus.ChunksToDelete = nil
```
* The cleanup on the caller function ca be done by initializing a new `UploadChunksStatusBody` struct abd only if there no errors returned from this function.

Reply:
done